index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+	<link href="registration.css" rel="stylesheet" type="text/css"> </head>
+
+<body>
+	<form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" name="test-form">
+		<fieldset class="personalInfo">
+			<legend>Personal information</legend>
+			<div class="form-field"> Surname:
+				<input type="text" name="surname" required> </div>
+			<div class="form-field"> Name:
+				<input type="text" name="name" required> </div>
+			<div class="form-field"> How old are you?
+				<input type="number" name="age"  minlength="1" maxlength="100" placeholder="12" required> </div>
+			<div class="form-field"> Full date of birth?
+				<input type="date" name="birthdate" placeholder="mm/dd/yyyy" required> </div>
```

Comment:
```suggestion
			<div class="form-field">
			  Full date of birth?
				<input
				  type="date"
				  name="birthdate"
				  placeholder="mm/dd/yyyy"
				  required
				>
				</div>
```

![image](https://user-images.githubusercontent.com/80700247/226168228-f1b232ec-570d-4ec2-8655-aa0cd72c5a96.png)